internal/btf/core.go

```diff
@@ -390,7 +399,23 @@ func coreCalculateFixup(local Type, localID TypeID, target Type, targetID TypeID
 			return fixup(uint32(localValue.Value), uint32(targetValue.Value))
 		}
 
-	case reloFieldByteOffset, reloFieldByteSize, reloFieldExists:
+	case reloFieldSigned:
```

Comment:
bitfields emit this (https://elixir.bootlin.com/linux/latest/source/tools/lib/bpf/bpf_core_read.h#L74). following the same approach as in libbpf (https://elixir.bootlin.com/linux/latest/source/tools/lib/bpf/relo_core.c#L657).